cmd/copyEnumeratorInit.go

```diff
@@ -112,9 +110,34 @@ func (cca *cookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 	// Giving it nothing to work with as new names will be added as we traverse.
 	var containerResolver = NewS3BucketNameToAzureResourcesResolver(nil)
 	existingContainers := make(map[string]bool)
-	var logDstContainerCreateFailureOnce sync.Once
 	seenFailedContainers := make(map[string]bool) // Create map of already failed container conversions so we don't log a million items just for one container.
 
+	createContainerFailureTransfer := func(dstContainerName, failureReason string) error {
```

Comment:
Minor: if you happen to be in here making any other changes, how about renaming this to `addContainerFailureTransfer` since the existing "create" name makes it sound like a factory method, and a factory method would not be expected to call `addTransfer` (whereas this does call `addTransfer`)